packages/bpk-docs/src/components/neo/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -59,9 +60,31 @@ class DocsPageWrapper extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = { selected: this.props.nativeSubpage ? 'native' : 'web' };
+
+    this.state = {
+      selected: this.props.nativeSubpage ? 'native' : 'web',
+    };
   }
 
+  componentWillMount = () => {
+    const urlParams = queryString.parse(window.location.search);
+    if (urlParams.platform) {
```

Comment:
I think we should validate this too, just to prevent any silly issues where platform is `foo` and then the page display gets messed up.